egs/voxceleb/v1/local/make_voxceleb1.pl

```diff
@@ -1,130 +1,127 @@
 #!/usr/bin/perl
 #
-# Copyright 2018  Ewald Enzinger
-#           2018  David Snyder
+# Copied from make_voxceleb2.pl and made necessary changes
 #
-# Usage: make_voxceleb1.pl /export/voxceleb1 data/
+# Usage: make_voxceleb1_v1.pl /export/voxceleb1 dev data/dev
+#
+# Note: This script requires ffmpeg to be installed and its location included in $PATH.
```

Comment:
Is ffmpeg actually required? It looks like you commented out some code related to that.